models/cxr_image_synthesis_latent_diffusion_model/configs/inference.json

```
@@ -0,0 +1,108 @@
+        "$from transformers import CLIPTextModel",
+        "$from transformers import CLIPTokenizer"
+    ],
+    "bundle_root": ".",
+    "load_old": 1,
+    "model_dir": "$@bundle_root + '/models'",
+    "output_dir": "$@bundle_root + '/output'",
+    "create_output_dir": "$Path(@output_dir).mkdir(exist_ok=True)",
+    "prompt": "Big right-sided pleural effusion",
+    "prompt_list": "$['', @prompt]",
+    "guidance_scale": 7.0,
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "tokenizer": "$CLIPTokenizer.from_pretrained(\"stabilityai/stable-diffusion-2-1-base\", subfolder=\"tokenizer\")",
+    "text_encoder": "$CLIPTextModel.from_pretrained(\"stabilityai/stable-diffusion-2-1-base\", subfolder=\"text_encoder\")",
+    "tokenized_prompt": "$@tokenizer(@prompt_list, padding=\"max_length\", max_length=@tokenizer.model_max_length, truncation=True,return_tensors=\"pt\")",
+    "prompt_embeds": "$@text_encoder(@tokenized_prompt.input_ids.squeeze(1))[0].to(@device)",
+    "out_file": "$datetime.now().strftime('sample_%H%M%S_%d%m%Y')",
+    "autoencoder_def": {
+        "_target_": "monai.networks.nets.AutoencoderKL",
+        "spatial_dims": 2,
+        "in_channels": 1,
+        "out_channels": 1,
+        "latent_channels": 3,
+        "channels": [
+            64,
+            128,
+            128,
+            128
+        ],
+        "num_res_blocks": 2,
+        "norm_num_groups": 32,
+        "norm_eps": 1e-06,
+        "attention_levels": [
+            false,
+            false,
+            false,
+            false
+        ],
+        "with_encoder_nonlocal_attn": false,
+        "with_decoder_nonlocal_attn": false
+    },
+    "load_autoencoder_path": "$@model_dir + '/autoencoder.pth'",
+    "load_autoencoder": "$@autoencoder_def.load_old_state_dict(torch.load(@load_autoencoder_path)) if bool(@load_old) else @autoencoder_def.load_state_dict(torch.load(@load_autoencoder_path))",
+    "autoencoder": "$@autoencoder_def.to(@device)",
+    "diffusion_def": {
+        "_target_": "monai.networks.nets.DiffusionModelUNet",
+        "spatial_dims": 2,
+        "in_channels": 3,
+        "out_channels": 3,
+        "channels": [
+            256,
+            512,
+            768
+        ],
+        "num_res_blocks": 2,
+        "attention_levels": [
+            false,
+            true,
+            true
+        ],
+        "norm_num_groups": 32,
+        "norm_eps": 1e-06,
+        "resblock_updown": false,
+        "num_head_channels": [
+            0,
+            512,
+            768
+        ],
+        "with_conditioning": true,
+        "transformer_num_layers": 1,
+        "cross_attention_dim": 1024
+    },
+    "load_diffusion_path": "$@model_dir + '/diffusion_model.pth'",
+    "load_diffusion": "$@diffusion_def.load_old_state_dict(torch.load(@load_diffusion_path)) if bool(@load_old) else @diffusion_def.load_state_dict(torch.load(@load_diffusion_path))",
```

```suggestion
    "load_diffusion_func": "$@diffusion_def.load_old_state_dict if bool(@load_old) else @diffusion_def.load_state_dict",
    "load_diffusion": "$@load_diffusion_func(torch.load(@load_diffusion_path))",
```
If you wanted to reduce the length of this line you can break things up into two values like this.